ocrd/ocrd/processor/helpers.py

```diff
@@ -163,6 +181,99 @@ def run_cli(
     result = run(args, check=False)
     return result.returncode
 
+
+async def run_cli_from_api(job_id: PydanticObjectId, executable: str, workspace, page_id: str,
+                           input_file_grps: List[str], output_file_grps: List[str], parameter: dict):
+    # Turn input/output file groups into a comma separated string
+    input_file_grps_str = ','.join(input_file_grps)
+    output_file_grps_str = ','.join(output_file_grps)
+
+    job = await Job.get(job_id)
+    job.state = StateEnum.running
+    job.start_time = datetime.now()
+
+    # Execute the processor
+    return_code = run_cli(executable, workspace=workspace, page_id=page_id, input_file_grp=input_file_grps_str,
+                          output_file_grp=output_file_grps_str, parameter=json.dumps(parameter), mets_url='')
```

Comment:
Hi, I think I maybe found a bug running a processing-server with ocrd-olena-binarize (this is a bashlib-processor). I think `mets_url` should be changed from `''` to `workspace.mets_target`, at least that is what resolves the problem in my case.
I think to reproduce it is enough to "simply" run the processing server with ocrd-olena-binarize and call it.